Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt

```diff
@@ -118,6 +118,15 @@
         <p tal:condition="not:portal_state/anonymous">Red titles indicate drafting-state items. Yellow titles indicate pending-state.</p>
 
         <div tal:condition="not:context/@@iterate/baseline|nothing">
+
+          <div id="representativePointField" class="field">
+            <label>Representative Point:</label>
+            <dl tal:replace="structure context/@@representative-point"></dl>
+            <div tal:condition="not:portal_state/anonymous">
```

Comment:
The "what's this" is in reference to the empty div with condition "not:portal_state/anonymous", which seems pointless. Note that most of the display code is repeated in a section where "baseline" is true in addition to the one where "baseline" is false. "baseline" is true when the current object is a working copy, and "baseline" is the original content object. This allows direct comparison of changed values on the view.